pom.xml

```diff
@@ -56,6 +56,7 @@
         <confluent.version>7.7.0</confluent.version>
         <!--Compatible protobuf version https://github.com/confluentinc/common/blob/v7.7.0/pom.xml#L91 -->
         <protobuf.version>3.25.5</protobuf.version>
+        <iceberg.version>1.5.2</iceberg.version>
```

Comment:
It's 1.6.1 in the ingest-sdk. Let's try to align.

Reply:
Done.